.github/workflows/android.yml

```diff
@@ -14,18 +14,46 @@ on:
       - '**.md'
 
 jobs:
+  test:
+    runs-on: macos-latest
+
+    steps:
+      - uses: actions/checkout@v3
+      - uses: dorny/paths-filter@v2
+        id: changes
+        with:
+          filters: |
+            ui:
+              - '**/ui/**'
```

Comment:
I also applied [paths-filter](https://github.com/dorny/paths-filter) to run androidTest conditionally.
The current setting in `on:` runs androidTest whenever a push or pull request occurs.
I added filter to check the location of updated files, the following tasks are started only if the path contains `ui`

* When updated files aren't in the ui package.
<img width="767" alt="스크린샷 2022-09-07 오전 12 01 20" src="https://user-images.githubusercontent.com/35446215/188671583-d15d543b-f262-4588-87b7-30c594fc0139.png">